rc/lsp.kak

```diff
@@ -967,6 +955,12 @@ method    = "textDocument/semanticTokens/full"
 ' "${kak_session}" "${kak_buffile}" "${kak_opt_filetype}" "${kak_timestamp}" | eval ${kak_opt_lsp_cmd} --request) > /dev/null 2>&1 < /dev/null & }
 }
 
+define-command -hidden lsp-semantic-tokens-refresh %{
+    evaluate-commands %sh{
+        printf 'evaluate-commands -client %s lsp-semantic-tokens-request\n' $kak_quoted_client_list
```

Comment:
Issue: if this is called even when semantic tokens isn't enabled by the user using hooks, then we'll get semantic tokens even though nothing was turned on.

I think the best solution to this is having a global toggle for semantic tokens on change and only actually doing anything in this callback when that's enabled.